src/plotman/manager.py

```diff
@@ -183,105 +183,108 @@ def key(key: str) -> job.Phase:
 
                     dstdir = max(dir2ph, key=key)
 
-            log_file_path = log_cfg.create_plot_log_path(time=pendulum.now())
-
-            plot_args: typing.List[str]
-            if plotting_cfg.type == "bladebit":
-                if plotting_cfg.bladebit is None:
-                    raise Exception(
-                        "bladebit plotter selected but not configured, report this as a plotman bug",
+            if plot_util.df_b(dstdir) > plot_util.get_plotsize(32):
```

Comment:
What if we invert this check and just return in that case and then avoid indenting everything else?

Reply:
Makes sense.  Have also added a check for tmp/dst and space required.

Reply:
re: your first comment - the newly freed space will be detected on a subsequent check (after the appropriate refresh time)?